### PR TITLE
fix: modify style of sms in settings

### PR DESCRIPTION
**Pull request by juanfranem**

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/settings/SyncManagerFragment.java [hunk 1]
```diff
@@ -230,14 +233,28 @@ public void showSmsSettings(boolean enabled, String number, boolean waitForRespo
...
+            ((TextInputLayout) binding.settingsSms.findViewById(R.id.settings_sms_receiver_layout)).setError(null);
+        } else if (!str.startsWith("+") && str.length() > 1) {
+            ((TextInputLayout) binding.settingsSms.findViewById(R.id.settings_sms_receiver_layout))
+                    .setError(binding.getRoot().getContext().getResources().getString(R.string.invalid_phone_number));
+            ((SwitchCompat) binding.settingsSms.findViewById(R.id.settings_sms_switch)).setChecked(false);
+        } else {
+            ((TextInputLayout) binding.settingsSms.findViewById(R.id.settings_sms_receiver_layout)).setError(null);
+        }
+        presenter.smsNumberSet(str);
+    }
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/settings/SyncManagerFragment.java]: can you move logic to the presenter and the call view methods ?
can you add unit tests?

**Comment by JaimeToca** [comment]
@ferdyrod @mmmateos can you check if sdk submodule is okay for this pr and help @juanfranem

**Comment by ferdyrod** [comment]
Added the correct commit of the sdk and rule-engine module

**Comment by ferdyrod** [comment]
@juanfranem Change the title of the PR to comply with the Semantic Pull Request.

**Comment by JaimeToca** [comment]
This PR is going to be closed. Please follow it in this link https://github.com/dhis2/dhis2-android-capture-app/pull/1111

**Pull request closed without merging**